<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1584</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="Controll">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1541</width>
     <height>851</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Main">
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>1171</width>
       <height>751</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Connecting to server...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_logout">
     <property name="geometry">
      <rect>
       <x>1340</x>
       <y>50</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1210</x>
       <y>40</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="btn_robot">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Robot</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_DB">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Database</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Reconnect</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>1210</x>
       <y>150</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="user_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>user_name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_user">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>User</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>1210</x>
       <y>260</y>
       <width>281</width>
       <height>521</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_score">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>score</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="LCD_score">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_speed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>speed</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="LCD_speed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>status</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_status_desc">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>211</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string>..</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_logout_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>470</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_lastPenalty">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">Color : red</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Log">
    <attribute name="title">
     <string>Log</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>1471</width>
       <height>741</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>datetime</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>score</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>minus reason</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>minus score</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>speed</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="object_log">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Object_Log</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>1471</width>
       <height>741</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>datetime</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>object</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
